esmvaltool/references/anav13jclim.bibtex

```diff
@@ -0,0 +1,13 @@
+@article{Anav_2013,
```

Comment:
Shouldn't we set this entry to:
```suggestion
@article{anav13jclim,
```
for consistency?

Reply:
Thanks @mattiarighi. This tag inside the bibtex file is generated automatically by doi2bib. There are more of this like: wang11climdyn.bibtex, williams09climdyn.bibtex, zhang11wcc.bibtex, ... . 
It seems that the doi2bib tags are not the same as our tags in config-references.yml.

Reply:
Yes, I know, the question is whether we could change that line and make it consistent with our tags. Also because duplicates may occur as our reference list grows.

Reply:
@mattiarighi thanks. The tags inside the bibtex files are now based on our tags. 
Please note that I also corrected some of the doi in the config-references.yml. However, I couldn't find the correct doi for two tags [davini18', 'sillman13jgr'].